proto/v1beta1/intoto.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+package grafeas.v1beta1.intoto;
+
+option go_package = "github.com/grafeas/grafeas/proto/v1beta1/intoto_go_proto";
+option java_multiple_files = true;
+option java_package = "io.grafeas.v1beta1.intoto";
+option objc_class_prefix = "GRA";
+
+import "google/protobuf/struct.proto";
+
+// This file defines the proto that matches the schema of an in-toto link
+// file. in-toto is a framework to protect a software supply chain. You can read
+// more about in-toto at https://in-toto.io/ and you can find the current
+// version of the in-toto specification at
+// https://github.com/in-toto/docs/blob/master/in-toto-spec.md
+
+
+// A signature object consists of the KeyID used and the signature itself.
+message Signature {
+  string keyid = 1 [json_name="keyid"];
+
+  string sig = 2 [json_name="sig"];
+}
+
+// An object defined to capture the by products of a step.
+message ByProductObject {
+  int32 return_value = 1 [json_name="return-value"];
+
+  string stderr = 2 [json_name="stderr"];
+
+  string stdout = 3 [json_name="stdout"];
+}
+
+// This corrresponds to the core in-toto link attestation.
+message Link {
+  // This field denotes the type of metadata being handled - in this case, the
+  // value is always "link".
+  string _type = 1 [json_name="_type"];
+
+  // This field contains the name of the step the link attestation was generated
+  // for.
+  string name = 2 [json_name="name"];
+
+  // This field contains the full command executed for the step. This can also
```

Comment:
"the full command executed for the step" -- is it expected that each part of the command will be a separate string in the repeated field? If so, it may be worth clarifying this and maybe adding an example

Reply:
Yes, each part of the command is a separate string. I'll include the example.

```"command": ["git","clone","https://github.com/in-toto/demo-project.git"]```